dts/bindings/ipc/zephyr,ipc-openamp-static-vrings.yaml

```diff
@@ -28,3 +28,18 @@ properties:
   mbox-names:
     description: MBOX channel names (must be called "tx" and "rx")
     required: true
+
+  priority:
+    type: array
+    required: false
+    description: |
+      WQ priority for the instance. This property is an array composed by the
+      priority level and the type of priority (PRIO_COOP for cooperative or
+      PRIO_PREEMPT for preemptive).
```

Comment:
I'm missing information if higher `priority level` value means a task with higher priority, or a task with lower priority, or how it translates to Zephyr's priority levels.

Reply:
It translates 1:1
```
priority = <1 PRIO_COOP> # == K_PRIO_COOP(1)

priority = <2 PRIO_PREEMPT> # == K_PRIO_PREEMPT(2)
```

Not sure how to be more clear that this.

Reply:
Can you add this to the documentation? I mean literally the example from your comment.